jiraprinter/jira.py

```diff
@@ -11,9 +11,11 @@
 import logging
 import bottle
 
+from jiraprinter.html.html_path import HTML_PATH
+
 logging.getLogger("requests").setLevel(logging.WARNING)
 
-DEFAULT_TEMPLATE = os.path.join(os.path.dirname(__file__), 'html', 'template.html')
+DEFAULT_TEMPLATE = os.path.join(HTML_PATH, 'template.html')
```

Comment:
You could use:

```python
from pkg_resources import resource_filename
DEFAULT_TEMPLATE = resource_filename('jiraprinter.html', 'template.html')
```

With that you don't need `html_path.py` at all.

Reply:
nah, I have to check out how it works xD